sentry-apache-http-client-5/build.gradle.kts

```diff
@@ -21,6 +21,8 @@ tasks.withType<KotlinCompile>().configureEach {
 dependencies {
     api(project(":sentry"))
     api(Config.Libs.apacheHttpClient)
+    api(Config.Libs.apacheHttpCore)
+    api(Config.Libs.apacheHttpCoreH2)
```

Comment:
do people need to access its classes? otherwise only `implementation` is fine

Reply:
They don't need to but they may want to. We bring `ApacheTransportFactory` but they may want to create their custom factory with custom client configuration. I am not 100% sure if at some level of configuration they would not need to use lower level classes.